tasks/github_tasks.py

```diff
@@ -579,3 +579,31 @@ def add_required_checks(_, branch: str, check: str, force: bool = False):
 
     gh = GithubAPI()
     gh.add_branch_required_check(branch, check, force)
+
+
+@task
+def check_qa_labels(_, labels_string: str):
+    """
+    Check if the PR has one of qa/[done|no-code-change|rc-required] label
+    """
+    labels = set(labels_string.split(" "))
```

Comment:
```suggestion
def check_qa_labels(_, labels: str):
    """
    Check if the PR has one of qa/[done|no-code-change|rc-required] label
    """
    labels = set(labels.split(" "))
```
If you agree with the above suggestion.